benchmark/hetero/count_if/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# Copyright Louis Dionne 2016
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
+
+set(datasets)
+add_dataset(datasets dataset.hetero.count_if.fusion.list   fusion.list.cpp.erb   "(0...50).step(5)")
+add_dataset(datasets dataset.hetero.count_if.fusion.vector fusion.vector.cpp.erb "(0...50).step(5)")
+add_dataset(datasets dataset.hetero.count_if.hana.tuple    hana.tuple.cpp.erb    "(0...50).step(5).to_a + (50..400).step(25).to_a")
```

Comment:
I think we should uniformize range across benchmarks, so that charts look somewhat similar.
I'd also be in favor of using a piecewise defined range in order to both reduce the time required to run the benchmark suite and also to increase the range, perhaps something like

`(0...99).step(10).to_a + (100...1000).step(50).to_a`